### DISCUSSION TITLE
How is blocklist supposed to function?

### DISCUSSION THREAD

**Discussion by chicknlil** [totaldebug/atomic-calendar-revive#845] [Support]
I have this card on a specific dashboard, and am utilizing two calendar feeds: my main email address, and a feed created from Todoist account. The Todoist list generates several items I don't want to appear on the dashboard, for example: 

`Clean off Feet from Previous Night`

Under the blacklist, I have the word `feet`, thinking this would be sufficient to remove this entry. However, it's not doing anything that I can tell (ie: this entry persists in my atomic calendar).

Anyone have any thoughts or suggestions how to accomplish my goals?

**Comment by Hanien1977** [comment]
Did you ever get this working? I am struggling with this now :)

**Reply by chicknlil** [reply]
Yes, I think it's case sensitive. So on one instance of ACR, I have this within the yaml:

`blocklist: Breakfast,Lunch,Dinner,Snack`

Hope this helps!

**Reply by marksie1988** [reply]
The card uses regex, you can test regex on regexr.com

**Reply by Hanien1977** [reply]
Thank you both, I found an example online and got it working :)
